### ISSUE TITLE
Dropdowns in navigation are always positioned from the left

### ISSUE THREAD

**Issue by bartaz** [canonical-web-and-design/vanilla-framework#2479]
Navigation dropdown are always positioned from left side of the toggle item.
When dropdown item is positioned on the right side of the page it makes dropdown expand to the right outside of the page.

It would be nice to have a modifier that would allow positioning the dropdown from the right to make it appear 'in page' even on far right side of the screen.

**Current**
<img width="317" alt="Screenshot 2019-08-13 at 08 30 34" src="https://user-images.githubusercontent.com/83575/62920105-00ab7180-bda5-11e9-871b-7237a3ad4b4e.png">

**Expected behavior**
<img width="324" alt="Screenshot 2019-08-13 at 08 30 52" src="https://user-images.githubusercontent.com/83575/62920112-04d78f00-bda5-11e9-8c77-fdf6f87feec6.png">

**Comment by lyubomir-popov** [comment]
To avoid conflicts, I'll wait for the theming spike pr to be merged before tackling this one.

**Comment by kwm14** [comment]
Theming spike [merged](https://github.com/canonical-web-and-design/vanilla-framework/pull/2507). Issue now unblocked 👍
